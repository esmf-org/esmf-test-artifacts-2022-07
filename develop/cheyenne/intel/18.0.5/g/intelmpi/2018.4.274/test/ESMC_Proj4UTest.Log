build time -- 2022-07-13 02:53:05.879139
20220713 032211.653 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220713 032211.653 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20220713 032211.653 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20220713 032211.653 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20220713 032211.653 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20220713 032211.653 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20220713 032211.653 INFO             PET0 Running with ESMF Version   : v8.4.0b03-3-g14a79d558f
20220713 032211.653 INFO             PET0 ESMF library build date/time: "Jul 13 2022" "02:46:33"
20220713 032211.653 INFO             PET0 ESMF library build location : /glade/scratch/dunlap/esmf-testing/intel_18.0.5_intelmpi_g_develop
20220713 032211.653 INFO             PET0 ESMF_COMM                   : intelmpi
20220713 032211.654 INFO             PET0 ESMF_MOAB                   : enabled
20220713 032211.654 INFO             PET0 ESMF_LAPACK                 : enabled
20220713 032211.654 INFO             PET0 ESMF_NETCDF                 : enabled
20220713 032211.654 INFO             PET0 ESMF_PNETCDF                : disabled
20220713 032211.654 INFO             PET0 ESMF_PIO                    : enabled
20220713 032211.654 INFO             PET0 ESMF_YAMLCPP                : enabled
20220713 032211.654 INFO             PET0 Beginning Test, file ESMC_Proj4UTest.C, line 49
20220713 032211.654 INFO             PET0 Beginning Test, file ESMC_Proj4UTest.C, line 49NUMBER_OF_PROCESSORS 1
20220713 032211.654 INFO             PET0 PASS PROJ.4 rotated pole transform test, ESMC_Proj4UTest.C, line 119
20220713 032211.654 INFO             PET0 Ending Test, file ESMC_Proj4UTest.C, line 123
20220713 032211.654 INFO             PET0 Finalizing ESMF
