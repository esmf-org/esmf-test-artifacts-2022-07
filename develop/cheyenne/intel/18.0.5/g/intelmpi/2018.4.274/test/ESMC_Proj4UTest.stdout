build time -- 2022-06-20 02:48:04.172896
Beginning Test, file ESMC_Proj4UTest.C, line 49
Beginning Test, file ESMC_Proj4UTest.C, line 49NUMBER_OF_PROCESSORS 1
PASS PROJ.4 rotated pole transform test, ESMC_Proj4UTest.C, line 119
Ending Test, file ESMC_Proj4UTest.C, line 123
Ending Test, file ESMC_Proj4UTest.C, line 123 PET 0 Test Elapsed Time 0.861 msec.
